image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
+*Reusing Images*
 
-=== Tagging images
-When tagging your own images, you should strive to maintain backwards compatibility within a tag.  For example, if you provide an image named "foo" and it currently includes version 1.0, you might provide a tag of "foo:v1".  When you update the image, as long as it continues to be compatible with the orginal image, you can continue to tag the new image "foo:v1" and downstream consumers of this tag will be able to get updates without being broken.  If you come out with an incompatible update, then you should switch to a new tag, e.g. "foo:v2".  This allows downstream consumers to move up to the new version at will, but not be inadvertently broken by the new incompatible image.
+Wherever possible, base your image on an appropriate upstream image using the `FROM` statement. This ensures your image can easily pick up security fixes from an upstream image when it is updated, rather than you being responsible for updating your dependencies directly.
 
-Of course any downstream consumer who chooses to use "foo:latest" will be taking on the risk of any incompatible changes introduced.
+In addition, use tags in the `FROM` instruction (for example,  `rhel:rhel7`) to make it clear to users exactly which version of an image your image is based on. Using a tag other than `latest` ensures your image is not subjected to breaking changes that might go into the `latest` version of an upstream image.
 
+*Tagging Images*
 
-=== Do not start multiple processes
-It is not a good idea to start multiple services (like database, sshd, …​) inside one container. Containers are lighweight and can be easily linked together for multiprocess orchestration.  OpenShift allows you to easily colocate and comanage related images by grouping them into a single pod.  This will ensure they share a network namespace and storage for communication.  It also makes updates less disruptive as each image can be updated less frequently and independently.  Having a single process also makes signal handling flows clearer as you do not need to manage routing signals to spawned processes.
+When tagging your own images, strive to maintain backwards compatibility within a tag. For example, if you provide an image named [sysitem]#foo# and it currently includes version 1.0, you might provide a tag of _foo:v1_. When you update the image, as long as it continues to be compatible with the original image, you can continue to tag the new image _foo:v1_, and downstream consumers of this tag will be able to get updates without being broken.
 
+If you later release an incompatible update, then you should switch to a new tag, for example _foo:v2_. This allows downstream consumers to move up to the new version at will, but not be inadvertently broken by the new incompatible image. Of course, any downstream consumer who chooses to use _foo:latest_ will be taking on the risk of any incompatible changes introduced.
 
```

Comment:
What can be "inadvertently broken by the new incompatible image" here? I think we should clarify this a bit more.